<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfiguerLiaisonGPS</class>
 <widget class="QDialog" name="DialogConfiguerLiaisonGPS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>341</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Choix du port :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxPort">
        <item>
         <property name="text">
          <string>Choissisez un port</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Vitesse de transmission :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxVitesse">
        <property name="currentText">
         <string>9600</string>
        </property>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nombre de bits de données :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxBitData">
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>8</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Parité</string>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>104</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="radioButtonPaire">
       <property name="text">
        <string>Paire</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonImpaire">
       <property name="text">
        <string>Impaire</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonRien">
       <property name="text">
        <string>Pas de parité</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBoxStop">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>131</y>
     <width>105</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1 bits de stop</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>166</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Control de flux :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxFlux">
      <item>
       <property name="text">
        <string>Pas de contrôle de flux</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Matériel (RTS/CTS)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Logiciel (XON/XOFF)</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogConfiguerLiaisonGPS</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogConfiguerLiaisonGPS</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
